enterprise/cypher/cypher/src/main/scala/org/neo4j/cypher/internal/compatibility/v3_3/runtime/interpreted/expressions/NodeFromRegister.scala

```diff
@@ -35,3 +35,15 @@ case class NodeFromRegister(offset: Int) extends Expression {
 
   override def symbolTableDependencies: Set[String] = Set.empty
 }
+
+case class ReferenceFromRegister(offset: Int) extends Expression {
```

Comment:
Why is this not in its own file? Does not really make sense in the NodeFromRegister.scala file